src/components/ProgressBar.jsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import styled from "styled-components";
+
+// 진행률 바 (0과 100일 때는 안 보이는 걸로 디자인)
+const ProgressBar = ({ progress }) => {
+  return (
+    <ProgressBarWrapper>
+      <Progress style={{ width: `${progress}%` }}>
+        {progress > 0 && progress < 100 && <Tooltip>{`${progress}%`}</Tooltip>}{" "}
+      </Progress>
+    </ProgressBarWrapper>
+  );
+};
```

Comment:
아주 아주 사소하지만 코드를 딱 봤을 때 한 눈에 알아보기 쉽게 로직을 살짝 분리해도 좋을 것 같아요!
```suggestion
const ProgressBar = ({ progress }) => {
  const isTodoing = progress > 0 && progress < 100;
  return (
    <ProgressBarWrapper>
      <Progress style={{ width: `${progress}%` }}>
        {isTodoing && <Tooltip>{`${progress}%`}</Tooltip>}
      </Progress>
    </ProgressBarWrapper>
  );
};
```